clearbit/prospector.go

```diff
@@ -48,7 +48,7 @@ type ProspectorService struct {
 func newProspectorService(sling *sling.Sling) *ProspectorService {
 	return &ProspectorService{
 		baseSling: sling.New(),
-		sling:     sling.Base(prospectorBase).Path("/v1/people/"),
+		sling:     sling.Base(prospectorBase).Path("/v1/people/").Set("Api-Version", "2016-10-04"),
```

Comment:
looks good. I might consider moving to const `const apiVersion = "2016-10-04"` at the top of the file
https://github.com/clearbit/clearbit-go/blob/c7b24c19d02ce16b60610210f3bbe574d9e2fec8/clearbit/prospector.go#L10